cfme/tests/cloud_infra_common/test_vm_ownership.py

```diff
@@ -182,6 +183,22 @@ def test_group_ownership_on_user_or_group_role(
         assert not check_vm_exists(vm_crud), "vm exists! but shouldn't exist"
 
 
+@pytest.mark.uncollectif(lambda provider: not provider.one_of(VMwareProvider),
+                         'Not VMware provider')
```

Comment:
**REQUIRED**
```python
@pytest.mark.provider([VMwareProvider])
```

Reply:
So this is a tricky one.

Every test in this module is currently marked by pytest.mark.provider([CloudInfraProvider], scope='module'), so I can't just mark this one with VMwareProvider, that doesn't work.

What I would have to do is mark each test individually with CloudInfraProvider except this one, which I'll mark with VMwareProvider, but every mark will have to be scope='module', because the fixtures that are used in the test are marked for module. So if I wanted to mark.provider with scope='function', it would produce a scope mismatch error. This seems to me a bit weird, marking each test inside a module with marker with scope='module'.

Another solution I can think of is just leave the uncollectif there, which solves all above issues.

What do you think?